src/App.tsx

```diff
@@ -15,71 +16,99 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+type State = {
+  selectedGood: string,
+  isSelected: boolean,
+};
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+export class App extends React.Component <{}, State> {
+  state = {
+    selectedGood: 'Jam',
+    isSelected: true,
```

Comment:
you don't need `isSelected` here, you can check whether the selected good is defined for such a purpose